clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -961,6 +960,75 @@ public String toRst() {
     }
 
     /**
+     * Get the configs with new metadata(group, orderInGroup, dependents) formatted with reStructuredText, suitable for embedding in Sphinx
+     * documentation.
+     */
+    public String toEnrichedRst() {
+        List<ConfigKey> configs = sortedConfigsInGroup();
+        StringBuilder b = new StringBuilder();
+        String lastKeyGroupName = "";
+
+        for (ConfigKey def : configs) {
+
+            if (!def.group.equalsIgnoreCase(lastKeyGroupName)) {
```

Comment:
I figured if you're calling `toEnrichedRst()` the new 0.10 fields are expected to be set